libraries/lib-wave-track/WaveClip.h

```diff
@@ -374,6 +370,9 @@ class WAVE_TRACK_API WaveClip final : public XMLTagHandler
    bool mIsPlaceholder { false };
 
 private:
+   //! Returns the index of the first sample of the underlying sequence
+   sampleCount GetSequenceStartSample() const;
```

Comment:
I'm starting the review in the middle, with the commit "Remove now obsolete API methods".  It makes the most widespread changes, by changing the interface of WaveTrack, and I want to understand that first as perhaps what motivates much else.

My first comment:  WaveClip::ToSequenceSamples becomes unused so could be removed.

Reply:
`ToSequenceSamples` can be removed, as now unused -- and then `GetSequenceStartSample` is also unused, so delete it too